apps/glusterfs/operations_block_volume.go

```diff
@@ -0,0 +1,353 @@
+//
+// Copyright (c) 2018 The heketi Authors
+//
+// This file is licensed to you under your choice of the GNU Lesser
+// General Public License, version 3 or any later version (LGPLv3 or
+// later), or the GNU General Public License, version 2 (GPLv2), in all
+// cases as published by the Free Software Foundation.
+//
+
+package glusterfs
+
+import (
+	"fmt"
+
+	"github.com/heketi/heketi/executors"
+	wdb "github.com/heketi/heketi/pkg/db"
+
+	"github.com/boltdb/bolt"
```

Comment:
this import order should be changed?

importing third party lib should be the second set of import

Reply:
Sure, but lets do that in a follow up commit. I've been trying to use that approach personally but right now heketi as a project does not enforce that as a general rule (yet).